var/spack/repos/builtin/packages/py-petsc4py/package.py

```diff
@@ -11,12 +11,13 @@ class PyPetsc4py(PythonPackage):
     """
 
     homepage = "https://gitlab.com/petsc/petsc4py"
-    url      = "https://gitlab.com/petsc/petsc4py/-/archive/3.13.0/petsc4py-3.13.0.tar.gz"
+    url      = "https://pypi.io/packages/source/p/petsc4py/petsc4py-3.14.0.tar.gz"
     git      = "https://gitlab.com/petsc/petsc4py.git"
 
     maintainers = ['dalcinl', 'balay']
 
     version('develop', branch='master')
```

Comment:
```suggestion
    version('master', branch='master')
```

Reply:
This affects other packages [wrt dependencies] so I'd rather not do this now [with xsdk changes]